src/components/plots/xyPlot.ts

```diff
@@ -154,19 +154,6 @@ module Plottable {
       return this;
     }
 
-    protected _propertyProjectors(): AttributeToProjector {
```

Comment:
It was used to set up the xProjector and the yProjector for pieces like `Plots.Bar` and `Drawers.Element`.  Now that the projectors are actually set up correctly, this override can finally go away.